google-cloud-clients/google-cloud-os-login/src/test/java/com/google/cloud/oslogin/v1/it/ITSystemTest.java

```diff
@@ -0,0 +1,100 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.oslogin.v1.it;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.api.gax.rpc.NotFoundException;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.oslogin.common.OsLoginProto;
+import com.google.cloud.oslogin.v1.FingerprintName;
+import com.google.cloud.oslogin.v1.ImportSshPublicKeyResponse;
+import com.google.cloud.oslogin.v1.LoginProfile;
+import com.google.cloud.oslogin.v1.OsLoginServiceClient;
+import com.google.cloud.oslogin.v1.UserName;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ITSystemTest {
+  private static OsLoginServiceClient osLoginServiceClient;
+  private static UserName parent;
+  private static String user;
+  private static String clientId;
+
+  private static final String PROJECT = ServiceOptions.getDefaultProjectId();
+  private static final String GOOGLE_API_CLIENT_ID = "clientId";
+  private static final String GOOGLE_API_CLIENT_EMAIL_ID = "clientEmail";
+  private static final String GOOGLE_API_CLOUD_PLATFORM_LINK =
+      "https://www.googleapis.com/auth/cloud-platform";
+  private static final String FINGERPRINT_NAME = "test-fingerprint";
+  private static final String REGEXP = "^\"|\"$";
+  private static final OsLoginProto.SshPublicKey SSH_PUBLIC_KEY =
+      OsLoginProto.SshPublicKey.newBuilder().build();
+  private static final Gson GSON = new Gson();
+
+  @BeforeClass
+  public static void beforeClass() throws Exception {
+    osLoginServiceClient = OsLoginServiceClient.create();
+    user = getFromCredential(GOOGLE_API_CLIENT_EMAIL_ID);
+    clientId = getFromCredential(GOOGLE_API_CLIENT_ID);
+    parent = UserName.of(user);
+  }
+
+  @AfterClass
+  public static void afterClass() {
+    osLoginServiceClient.close();
+  }
+
+  private static String getFromCredential(String key) throws Exception {
+    GoogleCredentials credentials =
+        GoogleCredentials.getApplicationDefault().createScoped(GOOGLE_API_CLOUD_PLATFORM_LINK);
+    JsonObject jsonObject = GSON.fromJson(GSON.toJson(credentials), JsonObject.class);
+    return jsonObject.get(key).toString().replaceAll(REGEXP, "");
+  }
+
+  @Test
+  public void getLoginProfileTest() {
+    LoginProfile profile = osLoginServiceClient.getLoginProfile(parent);
+    assertEquals(clientId, profile.getName());
+  }
+
+  @Test(expected = NotFoundException.class)
+  public void getSshPublicKeyExceptionTest() {
+    FingerprintName name = FingerprintName.of(user, FINGERPRINT_NAME);
+    OsLoginProto.SshPublicKey sshPublicKey = osLoginServiceClient.getSshPublicKey(name);
+  }
```

Comment:
Instead of expected attribute try something like this:
```java
  @Test
  public void getSshPublicKeyExceptionTest() {
    FingerprintName name = FingerprintName.of(user, FINGERPRINT_NAME);
    Exception actualException = null;
    try {
      OsLoginProto.SshPublicKey sshPublicKey = osLoginServiceClient.getSshPublicKey(name);
    } catch (Exception ex) {
      actualException = ex;
    }
    assertNotNull(actualException);
    assertTrue(actualException instanceof NotFoundException);
  }
```

Reply:
it's increase line of code at the end we get same result.

Reply:
Yes it is the same result but it's your call. I am fine with both the ways.